From 7b82b7f46272b00167f9c8634c932a38b1157243 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Mon, 9 May 2016 17:54:12 +0800
Subject: [PATCH 1258/5242] MLK-13441-1 ARM: imx7ulp: iomux: add imx7ulp
 header file

commit  20563b139b54848044b402cc7879cdb0fde4f3f6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add imx7ulp header file.

Two changes base on original header file from iomux tool team:
- Remove the mux register column since mux and conf is shared in
  one register.
- IOMUX_0 part:
  The register address: 0x4103_d000 ~ 0x4103_d0cc, the header file
  offset is 0xdxxx, now change it to 0x0xxx. The common base address
  0x4103_d000 is defined by dts file.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>

Renamed to same convention as upstream

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-pinfunc.h |  407 +++++++++++++++++++++++++++++++++++
 1 file changed, 407 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-pinfunc.h b/arch/arm/boot/dts/imx7ulp-pinfunc.h
index fe51177..2cd3602 100644
--- a/arch/arm/boot/dts/imx7ulp-pinfunc.h
+++ b/arch/arm/boot/dts/imx7ulp-pinfunc.h
@@ -16,6 +16,413 @@
  * <mux_conf_reg input_reg mux_mode input_val>
  */
 
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__CMP0_IN2A                         0x0000 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__PTA0                              0x0000 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__LLWU0_P0                          0x0000 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__LPSPI0_PCS1                       0x0000 0xd104 0x3 0x2
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__LPUART0_CTS_B                     0x0000 0xd1f8 0x4 0x2
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__LPI2C0_SCL                        0x0000 0xd17c 0x5 0x2
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__TPM0_CLKIN                        0x0000 0xd1a8 0x6 0x2
+#define IMX7ULP_PAD_PTA0_LLWU0_P0__I2S0_RX_BCLK                      0x0000 0x01b8 0x7 0x2
+#define IMX7ULP_PAD_PTA1__CMP0_IN2B                                  0x0004 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA1__PTA1                                       0x0004 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA1__LPSPI0_PCS2                                0x0004 0xd108 0x3 0x1
+#define IMX7ULP_PAD_PTA1__LPUART0_RTS_B                              0x0004 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA1__LPI2C0_SDA                                 0x0004 0xd180 0x5 0x1
+#define IMX7ULP_PAD_PTA1__TPM0_CH0                                   0x0004 0xd138 0x6 0x1
+#define IMX7ULP_PAD_PTA1__I2S0_RX_FS                                 0x0004 0x01bc 0x7 0x1
+#define IMX7ULP_PAD_PTA2__CMP1_IN2A                                  0x0008 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA2__PTA2                                       0x0008 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA2__LPSPI0_PCS3                                0x0008 0xd10c 0x3 0x1
+#define IMX7ULP_PAD_PTA2__LPUART0_TX                                 0x0008 0xd200 0x4 0x1
+#define IMX7ULP_PAD_PTA2__LPI2C0_HREQ                                0x0008 0xd178 0x5 0x1
+#define IMX7ULP_PAD_PTA2__TPM0_CH1                                   0x0008 0xd13c 0x6 0x1
+#define IMX7ULP_PAD_PTA2__I2S0_RXD0                                  0x0008 0x01dc 0x7 0x1
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__CMP1_IN2B                         0x000c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__PTA3                              0x000c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__CMP0_OUT                          0x000c 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__LLWU0_P1                          0x000c 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__LPUART0_RX                        0x000c 0xd1fc 0x4 0x1
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__TPM0_CH2                          0x000c 0xd140 0x6 0x1
+#define IMX7ULP_PAD_PTA3_LLWU0_P1__I2S0_RXD1                         0x000c 0x01e0 0x7 0x1
+#define IMX7ULP_PAD_PTA4__ADC1_CH2A                                  0x0010 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA4__PTA4                                       0x0010 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA4__LPSPI0_SIN                                 0x0010 0xd114 0x3 0x1
+#define IMX7ULP_PAD_PTA4__LPUART1_CTS_B                              0x0010 0xd204 0x4 0x1
+#define IMX7ULP_PAD_PTA4__LPI2C1_SCL                                 0x0010 0xd188 0x5 0x1
+#define IMX7ULP_PAD_PTA4__TPM0_CH3                                   0x0010 0xd144 0x6 0x1
+#define IMX7ULP_PAD_PTA4__I2S0_MCLK                                  0x0010 0x01b4 0x7 0x1
+#define IMX7ULP_PAD_PTA5__ADC1_CH2B                                  0x0014 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA5__PTA5                                       0x0014 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA5__LPSPI0_SOUT                                0x0014 0xd118 0x3 0x1
+#define IMX7ULP_PAD_PTA5__LPUART1_RTS_B                              0x0014 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA5__LPI2C1_SDA                                 0x0014 0xd18c 0x5 0x1
+#define IMX7ULP_PAD_PTA5__TPM0_CH4                                   0x0014 0xd148 0x6 0x1
+#define IMX7ULP_PAD_PTA5__I2S0_TX_BCLK                               0x0014 0x01c0 0x7 0x1
+#define IMX7ULP_PAD_PTA6__ADC1_CH3A                                  0x0018 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA6__PTA6                                       0x0018 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA6__LPSPI0_SCK                                 0x0018 0xd110 0x3 0x1
+#define IMX7ULP_PAD_PTA6__LPUART1_TX                                 0x0018 0xd20c 0x4 0x1
+#define IMX7ULP_PAD_PTA6__LPI2C1_HREQ                                0x0018 0xd184 0x5 0x1
+#define IMX7ULP_PAD_PTA6__TPM0_CH5                                   0x0018 0xd14c 0x6 0x1
+#define IMX7ULP_PAD_PTA6__I2S0_TX_FS                                 0x0018 0x01c4 0x7 0x1
+#define IMX7ULP_PAD_PTA7__ADC1_CH3B                                  0x001c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA7__PTA7                                       0x001c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA7__LPSPI0_PCS0                                0x001c 0xd100 0x3 0x1
+#define IMX7ULP_PAD_PTA7__LPUART1_RX                                 0x001c 0xd208 0x4 0x1
+#define IMX7ULP_PAD_PTA7__TPM1_CH1                                   0x001c 0xd154 0x6 0x1
+#define IMX7ULP_PAD_PTA7__I2S0_TXD0                                  0x001c 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA8__ADC1_CH7A                                  0x0020 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA8__PTA8                                       0x0020 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA8__LPSPI1_PCS1                                0x0020 0xd120 0x3 0x1
+#define IMX7ULP_PAD_PTA8__LPUART2_CTS_B                              0x0020 0xd210 0x4 0x1
+#define IMX7ULP_PAD_PTA8__LPI2C2_SCL                                 0x0020 0xd194 0x5 0x1
+#define IMX7ULP_PAD_PTA8__TPM1_CLKIN                                 0x0020 0xd1ac 0x6 0x1
+#define IMX7ULP_PAD_PTA8__I2S0_TXD1                                  0x0020 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA9__ADC1_CH7B                                  0x0024 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA9__PTA9                                       0x0024 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA9__NMI0_B                                     0x0024 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA9__LPSPI1_PCS2                                0x0024 0xd124 0x3 0x1
+#define IMX7ULP_PAD_PTA9__LPUART2_RTS_B                              0x0024 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA9__LPI2C2_SDA                                 0x0024 0xd198 0x5 0x1
+#define IMX7ULP_PAD_PTA9__TPM1_CH0                                   0x0024 0xd150 0x6 0x1
+#define IMX7ULP_PAD_PTA10__ADC1_CH6A                                 0x0028 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA10__PTA10                                     0x0028 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA10__LPSPI1_PCS3                               0x0028 0xd128 0x3 0x1
+#define IMX7ULP_PAD_PTA10__LPUART2_TX                                0x0028 0xd218 0x4 0x1
+#define IMX7ULP_PAD_PTA10__LPI2C2_HREQ                               0x0028 0xd190 0x5 0x1
+#define IMX7ULP_PAD_PTA10__TPM2_CLKIN                                0x0028 0xd1f4 0x6 0x1
+#define IMX7ULP_PAD_PTA10__I2S0_RX_BCLK                              0x0028 0x01b8 0x7 0x1
+#define IMX7ULP_PAD_PTA11__ADC1_CH6B                                 0x002c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA11__PTA11                                     0x002c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA11__LPUART2_RX                                0x002c 0xd214 0x4 0x1
+#define IMX7ULP_PAD_PTA11__TPM2_CH0                                  0x002c 0xd158 0x6 0x1
+#define IMX7ULP_PAD_PTA11__I2S0_RX_FS                                0x002c 0x01bc 0x7 0x2
+#define IMX7ULP_PAD_PTA12__ADC1_CH5A                                 0x0030 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA12__PTA12                                     0x0030 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA12__LPSPI1_SIN                                0x0030 0xd130 0x3 0x1
+#define IMX7ULP_PAD_PTA12__LPUART3_CTS_B                             0x0030 0xd21c 0x4 0x1
+#define IMX7ULP_PAD_PTA12__LPI2C3_SCL                                0x0030 0xd1a0 0x5 0x1
+#define IMX7ULP_PAD_PTA12__TPM2_CH1                                  0x0030 0xd15c 0x6 0x1
+#define IMX7ULP_PAD_PTA12__I2S0_RXD0                                 0x0030 0x01dc 0x7 0x2
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__ADC1_CH5B                        0x0034 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__PTA13                            0x0034 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__CMP0_OUT                         0x0034 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__LLWU0_P2                         0x0034 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__LPSPI1_SOUT                      0x0034 0xd134 0x3 0x2
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__LPUART3_RTS_B                    0x0034 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__LPI2C3_SDA                       0x0034 0xd1a4 0x5 0x2
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__TPM3_CLKIN                       0x0034 0xd1b0 0x6 0x1
+#define IMX7ULP_PAD_PTA13_LLWU0_P2__I2S0_RXD1                        0x0034 0x01e0 0x7 0x2
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__ADC1_CH4A                        0x0038 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__PTA14                            0x0038 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__LLWU0_P3                         0x0038 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__LPSPI1_SCK                       0x0038 0xd12c 0x3 0x2
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__LPUART3_TX                       0x0038 0xd224 0x4 0x2
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__LPI2C3_HREQ                      0x0038 0xd19c 0x5 0x2
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__TPM3_CH0                         0x0038 0xd160 0x6 0x1
+#define IMX7ULP_PAD_PTA14_LLWU0_P3__I2S0_MCLK                        0x0038 0x01b4 0x7 0x2
+#define IMX7ULP_PAD_PTA15__ADC1_CH4B                                 0x003c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA15__PTA15                                     0x003c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA15__LPSPI1_PCS0                               0x003c 0xd11c 0x3 0x1
+#define IMX7ULP_PAD_PTA15__LPUART3_RX                                0x003c 0xd220 0x4 0x1
+#define IMX7ULP_PAD_PTA15__TPM3_CH1                                  0x003c 0xd164 0x6 0x1
+#define IMX7ULP_PAD_PTA15__I2S0_TX_BCLK                              0x003c 0x01c0 0x7 0x2
+#define IMX7ULP_PAD_PTA16__CMP1_IN0A                                 0x0040 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA16__PTA16                                     0x0040 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA16__FXIO0_D0                                  0x0040 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA16__LPSPI0_PCS1                               0x0040 0xd104 0x3 0x1
+#define IMX7ULP_PAD_PTA16__LPUART0_CTS_B                             0x0040 0xd1f8 0x4 0x1
+#define IMX7ULP_PAD_PTA16__LPI2C0_SCL                                0x0040 0xd17c 0x5 0x1
+#define IMX7ULP_PAD_PTA16__TPM3_CH2                                  0x0040 0xd168 0x6 0x1
+#define IMX7ULP_PAD_PTA16__I2S0_TX_FS                                0x0040 0x01c4 0x7 0x2
+#define IMX7ULP_PAD_PTA17__CMP1_IN0B                                 0x0044 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA17__PTA17                                     0x0044 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA17__FXIO0_D1                                  0x0044 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA17__LPSPI0_PCS2                               0x0044 0xd108 0x3 0x2
+#define IMX7ULP_PAD_PTA17__LPUART0_RTS_B                             0x0044 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA17__LPI2C0_SDA                                0x0044 0xd180 0x5 0x2
+#define IMX7ULP_PAD_PTA17__TPM3_CH3                                  0x0044 0xd16c 0x6 0x1
+#define IMX7ULP_PAD_PTA17__I2S0_TXD0                                 0x0044 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__CMP1_IN1A                        0x0048 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__PTA18                            0x0048 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__NMI1_B                           0x0048 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__LLWU0_P4                         0x0048 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__FXIO0_D2                         0x0048 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__LPSPI0_PCS3                      0x0048 0xd10c 0x3 0x2
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__LPUART0_TX                       0x0048 0xd200 0x4 0x2
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__LPI2C0_HREQ                      0x0048 0xd178 0x5 0x2
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__TPM3_CH4                         0x0048 0xd170 0x6 0x1
+#define IMX7ULP_PAD_PTA18_LLWU0_P4__I2S0_TXD1                        0x0048 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__CMP1_IN1B                        0x004c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__PTA19                            0x004c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__LPTMR0_ALT3                      0x004c 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__LLWU0_P5                         0x004c 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__FXIO0_D3                         0x004c 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__LPUART0_RX                       0x004c 0xd1fc 0x4 0x2
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__TPM3_CH5                         0x004c 0xd174 0x6 0x1
+#define IMX7ULP_PAD_PTA19_LLWU0_P5__I2S1_RX_BCLK                     0x004c 0xd1cc 0x7 0x1
+#define IMX7ULP_PAD_PTA20__ADC0_CH7A                                 0x0050 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA20__PTA20                                     0x0050 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA20__FXIO0_D4                                  0x0050 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA20__LPSPI0_SIN                                0x0050 0xd114 0x3 0x2
+#define IMX7ULP_PAD_PTA20__LPUART1_CTS_B                             0x0050 0xd204 0x4 0x2
+#define IMX7ULP_PAD_PTA20__LPI2C1_SCL                                0x0050 0xd188 0x5 0x2
+#define IMX7ULP_PAD_PTA20__TPM0_CLKIN                                0x0050 0xd1a8 0x6 0x1
+#define IMX7ULP_PAD_PTA20__I2S1_RX_FS                                0x0050 0xd1d0 0x7 0x1
+#define IMX7ULP_PAD_PTA21__ADC0_CH7B                                 0x0054 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA21__PTA21                                     0x0054 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA21__FXIO0_D5                                  0x0054 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA21__LPSPI0_SOUT                               0x0054 0xd118 0x3 0x2
+#define IMX7ULP_PAD_PTA21__LPUART1_RTS_B                             0x0054 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA21__LPI2C1_SDA                                0x0054 0xd18c 0x5 0x2
+#define IMX7ULP_PAD_PTA21__TPM0_CH0                                  0x0054 0xd138 0x6 0x2
+#define IMX7ULP_PAD_PTA21__I2S1_RXD0                                 0x0054 0xd1e4 0x7 0x1
+#define IMX7ULP_PAD_PTA22__ADC0_CH6A                                 0x0058 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA22__PTA22                                     0x0058 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA22__LPTMR0_ALT2                               0x0058 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA22__EWM_OUT_B                                 0x0058 0x0000 0xc 0x0
+#define IMX7ULP_PAD_PTA22__FXIO0_D6                                  0x0058 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA22__LPSPI0_SCK                                0x0058 0xd110 0x3 0x2
+#define IMX7ULP_PAD_PTA22__LPUART1_TX                                0x0058 0xd20c 0x4 0x2
+#define IMX7ULP_PAD_PTA22__LPI2C1_HREQ                               0x0058 0xd184 0x5 0x2
+#define IMX7ULP_PAD_PTA22__TPM0_CH1                                  0x0058 0xd13c 0x6 0x2
+#define IMX7ULP_PAD_PTA22__I2S1_RXD1                                 0x0058 0xd1e8 0x7 0x1
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__ADC0_CH6B                        0x005c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__PTA23                            0x005c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__LLWU0_P6                         0x005c 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__FXIO0_D7                         0x005c 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__LPSPI0_PCS0                      0x005c 0xd100 0x3 0x2
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__LPUART1_RX                       0x005c 0xd208 0x4 0x2
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__TPM0_CH2                         0x005c 0xd140 0x6 0x2
+#define IMX7ULP_PAD_PTA23_LLWU0_P6__I2S1_MCLK                        0x005c 0xd1c8 0x7 0x1
+#define IMX7ULP_PAD_PTA24__ADC0_CH5A                                 0x0060 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA24__PTA24                                     0x0060 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA24__FXIO0_D8                                  0x0060 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA24__LPSPI1_PCS1                               0x0060 0xd120 0x3 0x2
+#define IMX7ULP_PAD_PTA24__LPUART2_CTS_B                             0x0060 0xd210 0x4 0x2
+#define IMX7ULP_PAD_PTA24__LPI2C2_SCL                                0x0060 0xd194 0x5 0x2
+#define IMX7ULP_PAD_PTA24__TPM0_CH3                                  0x0060 0xd144 0x6 0x2
+#define IMX7ULP_PAD_PTA24__I2S1_TX_BCLK                              0x0060 0xd1d4 0x7 0x1
+#define IMX7ULP_PAD_PTA25__ADC0_CH5B                                 0x0064 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA25__PTA25                                     0x0064 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA25__FXIO0_D9                                  0x0064 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA25__LPSPI1_PCS2                               0x0064 0xd124 0x3 0x2
+#define IMX7ULP_PAD_PTA25__LPUART2_RTS_B                             0x0064 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA25__LPI2C2_SDA                                0x0064 0xd198 0x5 0x2
+#define IMX7ULP_PAD_PTA25__TPM0_CH4                                  0x0064 0xd148 0x6 0x2
+#define IMX7ULP_PAD_PTA25__I2S1_TX_FS                                0x0064 0xd1d8 0x7 0x1
+#define IMX7ULP_PAD_PTA26__PTA26                                     0x0068 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA26__JTAG_TMS_SWD_DIO                          0x0068 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTA26__FXIO0_D10                                 0x0068 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA26__LPSPI1_PCS3                               0x0068 0xd128 0x3 0x2
+#define IMX7ULP_PAD_PTA26__LPUART2_TX                                0x0068 0xd218 0x4 0x2
+#define IMX7ULP_PAD_PTA26__LPI2C2_HREQ                               0x0068 0xd190 0x5 0x2
+#define IMX7ULP_PAD_PTA26__TPM0_CH5                                  0x0068 0xd14c 0x6 0x2
+#define IMX7ULP_PAD_PTA26__I2S1_RXD2                                 0x0068 0xd1ec 0x7 0x1
+#define IMX7ULP_PAD_PTA27__PTA27                                     0x006c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA27__JTAG_TDO                                  0x006c 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTA27__FXIO0_D11                                 0x006c 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA27__LPUART2_RX                                0x006c 0xd214 0x4 0x2
+#define IMX7ULP_PAD_PTA27__TPM1_CH1                                  0x006c 0xd154 0x6 0x2
+#define IMX7ULP_PAD_PTA27__I2S1_RXD3                                 0x006c 0xd1f0 0x7 0x1
+#define IMX7ULP_PAD_PTA28__PTA28                                     0x0070 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA28__JTAG_TDI                                  0x0070 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTA28__FXIO0_D12                                 0x0070 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA28__LPSPI1_SIN                                0x0070 0xd130 0x3 0x2
+#define IMX7ULP_PAD_PTA28__LPUART3_CTS_B                             0x0070 0xd21c 0x4 0x2
+#define IMX7ULP_PAD_PTA28__LPI2C3_SCL                                0x0070 0xd1a0 0x5 0x2
+#define IMX7ULP_PAD_PTA28__TPM1_CLKIN                                0x0070 0xd1ac 0x6 0x2
+#define IMX7ULP_PAD_PTA28__I2S1_TXD2                                 0x0070 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA29__PTA29                                     0x0074 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA29__JTAG_TCLK_SWD_CLK                         0x0074 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTA29__FXIO0_D13                                 0x0074 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA29__LPSPI1_SOUT                               0x0074 0xd134 0x3 0x1
+#define IMX7ULP_PAD_PTA29__LPUART3_RTS_B                             0x0074 0x0000 0x4 0x0
+#define IMX7ULP_PAD_PTA29__LPI2C3_SDA                                0x0074 0xd1a4 0x5 0x1
+#define IMX7ULP_PAD_PTA29__TPM1_CH0                                  0x0074 0xd150 0x6 0x2
+#define IMX7ULP_PAD_PTA29__I2S1_TXD3                                 0x0074 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA30__ADC0_CH4A                                 0x0078 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA30__PTA30                                     0x0078 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA30__JTAG_TRST_B                               0x0078 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTA30__FXIO0_D14                                 0x0078 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA30__LPSPI1_SCK                                0x0078 0xd12c 0x3 0x1
+#define IMX7ULP_PAD_PTA30__LPUART3_TX                                0x0078 0xd224 0x4 0x1
+#define IMX7ULP_PAD_PTA30__LPI2C3_HREQ                               0x0078 0xd19c 0x5 0x1
+#define IMX7ULP_PAD_PTA30__TPM2_CLKIN                                0x0078 0xd1f4 0x6 0x2
+#define IMX7ULP_PAD_PTA30__I2S1_TXD0                                 0x0078 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__ADC0_CH4B                        0x007c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__PTA31                            0x007c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__LPTMR0_ALT1                      0x007c 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__EWM_IN                           0x007c 0xd228 0xc 0x1
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__LLWU0_P7                         0x007c 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__FXIO0_D15                        0x007c 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__LPSPI1_PCS0                      0x007c 0xd11c 0x3 0x2
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__LPUART3_RX                       0x007c 0xd220 0x4 0x2
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__TPM2_CH0                         0x007c 0xd158 0x6 0x2
+#define IMX7ULP_PAD_PTA31_LLWU0_P7__I2S1_TXD1                        0x007c 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTB0__ADC0_CH0A                                  0x0080 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB0__PTB0                                       0x0080 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB0__CMP1_OUT                                   0x0080 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTB0__EWM_OUT_B                                  0x0080 0x0000 0xc 0x0
+#define IMX7ULP_PAD_PTB0__FXIO0_D16                                  0x0080 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB0__LPSPI0_SIN                                 0x0080 0xd114 0x3 0x3
+#define IMX7ULP_PAD_PTB0__LPUART0_TX                                 0x0080 0xd200 0x4 0x3
+#define IMX7ULP_PAD_PTB0__TPM2_CH1                                   0x0080 0xd15c 0x6 0x2
+#define IMX7ULP_PAD_PTB0__CLKOUT                                     0x0080 0x0000 0x9 0x0
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__ADC0_CH0B                         0x0084 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__PTB1                              0x0084 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__RTC_CLKOUT                        0x0084 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__EWM_IN                            0x0084 0xd228 0xc 0x2
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__LLWU0_P8                          0x0084 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__FXIO0_D17                         0x0084 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__LPSPI0_SOUT                       0x0084 0xd118 0x3 0x3
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__LPUART0_RX                        0x0084 0xd1fc 0x4 0x3
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__TPM3_CLKIN                        0x0084 0xd1b0 0x6 0x3
+#define IMX7ULP_PAD_PTB1_LLWU0_P8__I2S1_TX_BCLK                      0x0084 0xd1d4 0x7 0x2
+#define IMX7ULP_PAD_PTB2__ADC0_CH1A                                  0x0088 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB2__PTB2                                       0x0088 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB2__TRACE_CLKOUT                               0x0088 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB2__FXIO0_D18                                  0x0088 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB2__LPSPI0_SCK                                 0x0088 0xd110 0x3 0x3
+#define IMX7ULP_PAD_PTB2__LPUART1_TX                                 0x0088 0xd20c 0x4 0x3
+#define IMX7ULP_PAD_PTB2__TPM3_CH0                                   0x0088 0xd160 0x6 0x2
+#define IMX7ULP_PAD_PTB2__I2S1_TX_FS                                 0x0088 0xd1d8 0x7 0x2
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__ADC0_CH1B                         0x008c 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__PTB3                              0x008c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__TRACE_D0                          0x008c 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__LPTMR1_ALT2                       0x008c 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__LLWU0_P9                          0x008c 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__FXIO0_D19                         0x008c 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__LPSPI0_PCS0                       0x008c 0xd100 0x3 0x3
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__LPUART1_RX                        0x008c 0xd208 0x4 0x3
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__TPM3_CH1                          0x008c 0xd164 0x6 0x2
+#define IMX7ULP_PAD_PTB3_LLWU0_P9__I2S1_TXD0                         0x008c 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTB4__PTB4                                       0x0090 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB4__TRACE_D1                                   0x0090 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB4__BOOTCFG0                                   0x0090 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB4__FXIO0_D20                                  0x0090 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB4__LPSPI0_PCS1                                0x0090 0xd104 0x3 0x3
+#define IMX7ULP_PAD_PTB4__LPUART2_TX                                 0x0090 0xd218 0x4 0x3
+#define IMX7ULP_PAD_PTB4__LPI2C0_HREQ                                0x0090 0xd178 0x5 0x3
+#define IMX7ULP_PAD_PTB4__TPM3_CH2                                   0x0090 0xd168 0x6 0x2
+#define IMX7ULP_PAD_PTB4__I2S1_TXD1                                  0x0090 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTB5__PTB5                                       0x0094 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB5__TRACE_D2                                   0x0094 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB5__BOOTCFG1                                   0x0094 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB5__FXIO0_D21                                  0x0094 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB5__LPSPI0_PCS2                                0x0094 0xd108 0x3 0x3
+#define IMX7ULP_PAD_PTB5__LPUART2_RX                                 0x0094 0xd214 0x4 0x3
+#define IMX7ULP_PAD_PTB5__LPI2C1_HREQ                                0x0094 0xd184 0x5 0x3
+#define IMX7ULP_PAD_PTB5__TPM3_CH3                                   0x0094 0xd16c 0x6 0x2
+#define IMX7ULP_PAD_PTB5__I2S1_TXD2                                  0x0094 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__PTB6                             0x0098 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__TRACE_D3                         0x0098 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__LPTMR1_ALT3                      0x0098 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__LLWU0_P10                        0x0098 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__FXIO0_D22                        0x0098 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__LPSPI0_PCS3                      0x0098 0xd10c 0x3 0x3
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__LPUART3_TX                       0x0098 0xd224 0x4 0x3
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__LPI2C0_SCL                       0x0098 0xd17c 0x5 0x3
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__TPM3_CH4                         0x0098 0xd170 0x6 0x2
+#define IMX7ULP_PAD_PTB6_LLWU0_P10__I2S1_TXD3                        0x0098 0x0000 0x7 0x0
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__PTB7                             0x009c 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__CMP1_OUT                         0x009c 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__LLWU0_P11                        0x009c 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__FXIO0_D23                        0x009c 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__LPSPI1_SIN                       0x009c 0xd130 0x3 0x3
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__LPUART3_RX                       0x009c 0xd220 0x4 0x3
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__LPI2C0_SDA                       0x009c 0xd180 0x5 0x3
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__TPM3_CH5                         0x009c 0xd174 0x6 0x2
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__I2S1_MCLK                        0x009c 0xd1c8 0x7 0x2
+#define IMX7ULP_PAD_PTB7_LLWU0_P11__QSPIA_SS1_B                      0x009c 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB8__CMP0_IN0A                                  0x00a0 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB8__PTB8                                       0x00a0 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB8__RTC_CLKOUT                                 0x00a0 0x0000 0xb 0x0
+#define IMX7ULP_PAD_PTB8__FXIO0_D24                                  0x00a0 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB8__LPSPI1_SOUT                                0x00a0 0xd134 0x3 0x3
+#define IMX7ULP_PAD_PTB8__LPI2C1_SCL                                 0x00a0 0xd188 0x5 0x3
+#define IMX7ULP_PAD_PTB8__TPM0_CLKIN                                 0x00a0 0xd1a8 0x6 0x3
+#define IMX7ULP_PAD_PTB8__I2S1_RX_BCLK                               0x00a0 0xd1cc 0x7 0x2
+#define IMX7ULP_PAD_PTB8__QSPIA_SS0_B                                0x00a0 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__CMP0_IN0B                        0x00a4 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__PTB9                             0x00a4 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__LLWU0_P12                        0x00a4 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__FXIO0_D25                        0x00a4 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__LPSPI1_SCK                       0x00a4 0xd12c 0x3 0x3
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__LPI2C1_SDA                       0x00a4 0xd18c 0x5 0x3
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__TPM0_CH0                         0x00a4 0xd138 0x6 0x3
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__I2S1_RX_FS                       0x00a4 0xd1d0 0x7 0x2
+#define IMX7ULP_PAD_PTB9_LLWU0_P12__QSPIA_DQS                        0x00a4 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB10__CMP0_IN1A                                 0x00a8 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB10__PTB10                                     0x00a8 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB10__TRACE_D4                                  0x00a8 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB10__FXIO0_D26                                 0x00a8 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB10__LPSPI1_PCS0                               0x00a8 0xd11c 0x3 0x3
+#define IMX7ULP_PAD_PTB10__LPI2C2_SCL                                0x00a8 0xd194 0x5 0x3
+#define IMX7ULP_PAD_PTB10__TPM0_CH1                                  0x00a8 0xd13c 0x6 0x3
+#define IMX7ULP_PAD_PTB10__I2S1_RXD0                                 0x00a8 0xd1e4 0x7 0x2
+#define IMX7ULP_PAD_PTB10__QSPIA_DATA7                               0x00a8 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB11__CMP0_IN1B                                 0x00ac 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB11__PTB11                                     0x00ac 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB11__TRACE_D5                                  0x00ac 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB11__FXIO0_D27                                 0x00ac 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB11__LPSPI1_PCS1                               0x00ac 0xd120 0x3 0x3
+#define IMX7ULP_PAD_PTB11__LPI2C2_SDA                                0x00ac 0xd198 0x5 0x3
+#define IMX7ULP_PAD_PTB11__TPM1_CLKIN                                0x00ac 0xd1ac 0x6 0x3
+#define IMX7ULP_PAD_PTB11__I2S1_RXD1                                 0x00ac 0xd1e8 0x7 0x2
+#define IMX7ULP_PAD_PTB11__QSPIA_DATA6                               0x00ac 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB12__ADC1_CH0A                                 0x00b0 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB12__PTB12                                     0x00b0 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB12__TRACE_D6                                  0x00b0 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB12__FXIO0_D28                                 0x00b0 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB12__LPSPI1_PCS2                               0x00b0 0xd124 0x3 0x3
+#define IMX7ULP_PAD_PTB12__LPI2C3_SCL                                0x00b0 0xd1a0 0x5 0x3
+#define IMX7ULP_PAD_PTB12__TPM1_CH0                                  0x00b0 0xd150 0x6 0x3
+#define IMX7ULP_PAD_PTB12__I2S1_RXD2                                 0x00b0 0xd1ec 0x7 0x2
+#define IMX7ULP_PAD_PTB12__QSPIA_DATA5                               0x00b0 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB13__ADC1_CH0B                                 0x00b4 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB13__PTB13                                     0x00b4 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB13__TRACE_D7                                  0x00b4 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB13__FXIO0_D29                                 0x00b4 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB13__LPSPI1_PCS3                               0x00b4 0xd128 0x3 0x3
+#define IMX7ULP_PAD_PTB13__LPI2C3_SDA                                0x00b4 0xd1a4 0x5 0x3
+#define IMX7ULP_PAD_PTB13__TPM1_CH1                                  0x00b4 0xd154 0x6 0x3
+#define IMX7ULP_PAD_PTB13__I2S1_RXD3                                 0x00b4 0xd1f0 0x7 0x2
+#define IMX7ULP_PAD_PTB13__QSPIA_DATA4                               0x00b4 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__ADC1_CH1A                       0x00b8 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__PTB14                           0x00b8 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__LLWU0_P13                       0x00b8 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__FXIO0_D30                       0x00b8 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__LPI2C2_HREQ                     0x00b8 0xd190 0x5 0x3
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__TPM2_CLKIN                      0x00b8 0xd1f4 0x6 0x3
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__QSPIA_SS0_B                     0x00b8 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB14_LLWU0_P13__QSPIA_SCLK_B                    0x00b8 0x0000 0x9 0x0
+#define IMX7ULP_PAD_PTB15__ADC1_CH1B                                 0x00bc 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB15__PTB15                                     0x00bc 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB15__FXIO0_D31                                 0x00bc 0x0000 0x2 0x0
+#define IMX7ULP_PAD_PTB15__LPI2C3_HREQ                               0x00bc 0xd19c 0x5 0x3
+#define IMX7ULP_PAD_PTB15__TPM2_CH0                                  0x00bc 0xd158 0x6 0x3
+#define IMX7ULP_PAD_PTB15__QSPIA_SCLK                                0x00bc 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB16_LLWU0_P14__ADC0_CH2A                       0x00c0 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB16_LLWU0_P14__PTB16                           0x00c0 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB16_LLWU0_P14__LLWU0_P14                       0x00c0 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB16_LLWU0_P14__TPM2_CH1                        0x00c0 0xd15c 0x6 0x3
+#define IMX7ULP_PAD_PTB16_LLWU0_P14__QSPIA_DATA3                     0x00c0 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB17__ADC0_CH2B                                 0x00c4 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB17__PTB17                                     0x00c4 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB17__TPM3_CLKIN                                0x00c4 0xd1b0 0x6 0x2
+#define IMX7ULP_PAD_PTB17__QSPIA_DATA2                               0x00c4 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB18__ADC0_CH3A                                 0x00c8 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB18__PTB18                                     0x00c8 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB18__TPM3_CH0                                  0x00c8 0xd160 0x6 0x3
+#define IMX7ULP_PAD_PTB18__QSPIA_DATA1                               0x00c8 0x0000 0x8 0x0
+#define IMX7ULP_PAD_PTB19_LLWU0_P15__ADC0_CH3B                       0x00cc 0x0000 0x0 0x0
+#define IMX7ULP_PAD_PTB19_LLWU0_P15__PTB19                           0x00cc 0x0000 0x1 0x0
+#define IMX7ULP_PAD_PTB19_LLWU0_P15__USB0_ID                         0x00cc 0x0000 0xa 0x0
+#define IMX7ULP_PAD_PTB19_LLWU0_P15__LLWU0_P15                       0x00cc 0x0000 0xd 0x0
+#define IMX7ULP_PAD_PTB19_LLWU0_P15__TPM3_CH1                        0x00cc 0xd164 0x6 0x3
+#define IMX7ULP_PAD_PTB19_LLWU0_P15__QSPIA_DATA0                     0x00cc 0x0000 0x8 0x0
 #define IMX7ULP_PAD_PTC0__PTC0                                       0x0000 0x0000 0x1 0x0
 #define IMX7ULP_PAD_PTC0__TRACE_D15                                  0x0000 0x0000 0xa 0x0
 #define IMX7ULP_PAD_PTC0__LPUART4_CTS_B                              0x0000 0x0244 0x4 0x1
-- 
1.7.9.5

